spec/factories/digital_object_imports.rb

```diff
@@ -5,6 +5,7 @@
     index { 34 }
     digital_object_data do
       {
+        'assign_uid' => '2f4e2917-26f5-4d8f-968c-a4015b10e50f',
```

Comment:
Rather than adding assign_uid to the base factory definition, it would be better to create a new trait like `:item_with_assign_uid` so that assign_uid doesn't appear as the default case (since, upcoming migration scenario aside, it will be infrequent).  I think this will also make it less necessary to modify existing text fixture files (e.g. sample-batch-import.csv).